Lib/fontTools/feaLib/parser.py

```diff
@@ -712,6 +714,77 @@ def parse_table_head_(self, table):
                 raise FeatureLibError("Expected FontRevision",
                                       self.cur_token_location_)
 
+    def parse_table_name_(self, table):
+        statements = table.statements
+        while self.next_token_ != "}":
+            self.advance_lexer_()
+            if self.is_cur_keyword_("nameid"):
+                statement = self.parse_nameid_()
+                if statement:
+                    statements.append(statement)
+            else:
+                raise FeatureLibError("Expected nameid",
+                                      self.cur_token_location_)
+
+    def parse_nameid_(self):
+        assert self.cur_token_ == "nameid", self.cur_token_
+        location, nameID = self.cur_token_location_, self.expect_number_()
+        if nameID > 255:
+            raise FeatureLibError("Expected name id < 255",
+                                  self.cur_token_location_)
+        if 1 <= nameID <= 6:
+            # TODO: issue a warning
```

Comment:
I suggest you import `logging` and create a `log` instance at the top of the parser module:

```python
import logging
log = logging.getLogger(__name__)
```

and then here you call `log.warning(...)` to send an appropriate warning message.